lib/Doctrine/Node/NestedSet/PreOrderIterator.php

```diff
@@ -67,6 +67,12 @@ class Doctrine_Node_NestedSet_PreOrderIterator implements Iterator
      */
     protected $count;
 
+    // These were undefined, added for static analysis and set to public so api isn't changed
```

Comment:
These seem to only ever be used locally so that the two methods of this class can share the values.

I confirmed that this doesn't extend anything that uses __get/__call.